### PR TITLE
New Client ID for Google Login and Fix for Search feature

### PR DESCRIPTION
**Pull request by MauroGuerrero20**
- Required in order to deploy to sopa-capstone-step-2020.
- Also fixes the search bar in deployment

### CODE REVIEW

**Review by steven-solar — APPROVED** [review]
LGTM

**Review by geoffbass — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/Navbar.jsx [hunk 1]
```diff
@@ -56,8 +56,8 @@ class Navbar extends Component {
         </Link>
         <div className={navClassOne} id="navbarNavDropdown">
           <ul className="navbar-nav">
-            <form className="form-inline my-2 my-lg-0">
-              <input className="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search" />
+            <form className="form-inline my-2 my-lg-0" onSubmit={this.handleSearchInput}>
```
Comment (antoniolinhart) [inline capstone-frontend/src/components/Navbar.jsx:59]: Thanks for fixing this!

**Review by antoniolinhart — APPROVED** [review]
LGTM

**Pull request merged**